controllers/reservation.controllers.js

```diff
@@ -83,47 +81,473 @@ const listReservationByDate = async ({ client, ack, say, body }) => {
+      }
+    } else {
+      await client.views.open({
+        // Pass a valid trigger_id within 3 seconds of receiving it
+        trigger_id: body.trigger_id,
+        // View payload
+        view: {
+          callback_id: 'add_reserve',
+          type: 'modal',
+          title: {
+            type: 'plain_text',
+            text: 'Reserva tu lugar',
+            emoji: true,
+          },
+          submit: {
+            type: 'plain_text',
+            text: 'Reservar',
+            emoji: true,
+          },
+          close: {
+            type: 'plain_text',
+            text: 'Cancelar',
+            emoji: true,
+          },
+          blocks: [
+            {
+              type: 'input',
+              block_id: 'date_input',
+              element: {
+                type: 'datepicker',
+                initial_date: new Date().toISOString().split('T')[0],
+                action_id: 'datepicker-action',
+              },
+              label: {
+                type: 'plain_text',
+                text: 'Cuando vas a la oficina?',
+                emoji: true,
+              },
+            },
+            {
+              type: 'input',
+              block_id: 'frecuency_input',
+              element: {
+                type: 'radio_buttons',
+                options: [
+                  {
+                    text: {
+                      type: 'plain_text',
+                      text: 'Solo este día',
+                      emoji: true,
+                    },
+                    value: 'day',
+                  },
+                  {
+                    text: {
+                      type: 'plain_text',
+                      text: 'Esta semana',
+                      emoji: true,
+                    },
+                    value: 'week',
+                  },
+                  {
+                    text: {
+                      type: 'plain_text',
+                      text: 'Este mes',
+                      emoji: true,
+                    },
+                    value: 'month',
+                  },
+                ],
+                action_id: 'radio_button-action',
+              },
+              label: {
+                type: 'plain_text',
+                text: 'Selecciona un lugar',
+                emoji: true,
+              },
+            },
+            {
+              type: 'input',
+              block_id: 'site_input',
+              element: {
+                type: 'static_select',
+                placeholder: {
+                  type: 'plain_text',
+                  text: 'Selecciona un lugar',
+                  emoji: true,
+                },
+                options,
+                action_id: 'static_select-action',
+              },
+              label: {
+                type: 'plain_text',
+                text: 'Label',
+                emoji: true,
+              },
+            },
+          ],
+        },
+      });
+    }
+  } catch (error) {
+    say(error);
```

Comment:
most errors look the same, you could throw errors and only do the `say` here in the catch